internal/cmd/beta/network/list/list.go

```diff
@@ -151,15 +151,20 @@ func outputResult(p *print.Printer, outputFormat string, networks []iaas.Network
 		return nil
 	default:
 		table := tables.NewTable()
-		table.SetHeader("ID", "Name", "Status", "Public IP")
+		table.SetHeader("ID", "NAME", "STATUS", "PUBLIC IP", "ROUTED")
 
 		for _, network := range networks {
 			publicIp := ""
 			if network.PublicIp != nil {
 				publicIp = *network.PublicIp
 			}
 
-			table.AddRow(*network.NetworkId, *network.Name, *network.State, publicIp)
+			routed := false
+			if network.Routed != nil {
+				routed = *network.Routed
+			}
```

Comment:
:+1:

Reply:
This same handling could also be applied to the output of the `describe`, right?

Reply:
I adapted the routed row handling